informatics-isi-edu/chaise
# Implemented new window concept for Login

This PR fixes issue https://github.com/informatics-isi-edu/chaise/issues/982.

Whenever a 401 Unauthorized error is encountered during add/edit/delete  a login window will open. Once you're done submitting your credentials, it will close and focus back on the main tab, restarting the submit automatically.

https://dev.gpcrconsortium.org/~chirag/chaise/recordedit/#1/experiments:sample
To test the flow

-Open above link and fill up the form
-Now open the same link in another tab and press logout
-Now go back to previous page where you had filled the form and click submit
-You should be presented with a login window and on successful login you should be able to create/edit/delete the record.

The code handles non-globus authentication flow too
